### DISCUSSION TITLE
Welcome to the AI in Biology class discussions!

### DISCUSSION THREAD

**Discussion by magitz** [AIBiology/aibiology.github.io#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other students and the instructors. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

## Class Demeanor and Netiquette
The discussion boards can be used to ask for and provide help by all. Students should be supportive and considerate of others at all times. Rude or inappropriate comments will be removed and the poster will be warned.

## Getting started
To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AadavanElangovan** [comment]
Is codespaces down for anybody else right now? It is for me, and I might be screwed on submitting my assignment.
